README.md

```diff
@@ -40,103 +42,117 @@ We will use the same project structure we used for the previous project. Classes
 1. Create the class for the object tested in the step above (**GREEN**)
 1. Use `git add`, `commit` and `push` commands to push your initial code to GitHub
 
-### Process
+## Process Requirements
 
 You should use the following process as much as possible:
 
 1. Write pseudocode
 1. Write test(s)
 1. Write code
+1. Refactor
 1. Commit
 
-You should have **95% code coverage** using `simplecov`.
-
 Your git commit history should provide a clear description of how your code developed, letting the reader know what changed when and why. Making frequent, small commits is essential!
 
-## Wave Zero: Project Design
+### Wave Zero: In-Class Design Activity
+
+Before starting work on the functional requirements, create a first draft of your project's design in the in-class instructor-led activity.
+
+## Functional Requirements
 
-This will be an in-class activity.
+### Wave One: Tracking Reservations
 
-## Wave One: Tracking Reservations
+In this wave, write the functionality for the system to track valid reservations, so that a user of the hotel system can make and find valid bookings for their hotel.
 
-Remember that your job is only to build the classes that store information and handle business logic, and the tests to verify they're behaving as expected. Building a user interface is not part of this project!
+Remember: Your job is to only build the classes that store information and handle business logic, and the tests to verify they're behaving as expected. Building a user interface is not part of this project!
 
-### User Stories
+#### User Stories
 
-- As an administrator, I can access the list of all of the rooms in the hotel
-- As an administrator, I can reserve a room for a given date range
-- As an administrator, I can access the list of reservations for a specific date
-- As an administrator, I can get the total cost for a given reservation
+- As a user of the hotel system...
+  - I can access the list of all of the rooms in the hotel
+  - I can reserve a room for a given date range, so that I can make a reservation
+  - I can access the list of reservations for a specific date, so that I can track reservations by date
+  - I can get the total cost for a given reservation
+  - I want a [StandardError](https://ruby-doc.org/core-2.5.0/StandardError.html) raised when an invalid date range is provided, so that I can't make a reservation for an invalid date range
 
```

Comment:
If you remove the `-2.5.0` from this address, you get a link to the most recent version of the docs.

Reply:
Also, they know about custom exceptions already! We should not encourage them to `raise StandardError` directly, that's a bad habit. We should be asking them to define custom exception types instead.